snapshot/README.md

```diff
@@ -0,0 +1,139 @@
+--- 
+layout: pattern
+title: Snapshot
+folder: snapshot
+permalink: /patterns/snapshot/
+categories:
+- creational 
+  language: 
+  tags:
+- Data access
+- Gang of Four
+---
+
+## Name / classification
+
+Snapshot.
+
+## Intent
+
+Create a view of an object at a point in time.
+
+## Explanation
+
+Real world example
+
+> Many people have phone numbers with details attached. A phone book, where registered phone 
+> numbers with details are relieved are snapshots of the history of phone numbers, and their 
+> associated details.
```

Comment:
What is meant by "where registered phone numbers with detailed are relieved are snapshots"? I find this a bit difficult to read.